glide-core/src/client/value_conversion.rs

```diff
@@ -752,6 +780,14 @@ mod tests {
         assert_eq!(expected_response, converted_flat_array);
     }
 
+    #[test]
+    fn convert_to_array_of_string_and_array_return_type() {
+        assert!(matches!(
+            expected_type_for_cmd(redis::cmd("LMPOP").arg("1").arg("key").arg("LEFT")),
```

Comment:
`LMPOP` doesn't exist here since that PR is still receiving updates and I chose to not base this change off of that one, to have an easier time with the commit history, hence the rust check is failing.